frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -0,0 +1,203 @@
+                item-value="facilityId"></v-select>
+            </v-col>
+          </v-row>
+          <v-row no-gutters align="center" class="mt-2">
+            <div class="blueText mr-8">
+              <strong>Supporting documents</strong>
+              (optional):
+            </div>
+            <AppButton id="upload-document" :isPrimary="false" @click="false">
+              <v-icon class="mr-3">mdi-tray-arrow-up</v-icon>
+              Upload
+            </AppButton>
+          </v-row>
+          <v-row no-gutters class="mt-2">
+            <v-col class="v-col-12 blueText pb-0">
+              <strong>Preferred method of contact:</strong>
+            </v-col>
+            <v-col class="v-col-12">
+              <v-radio-group v-model="newRequestModel.contactMethod" :rules="rules.required" inline color="primary">
+                <v-radio label="Phone" value="2"></v-radio>
+                <v-radio label="Portal message" value="1"></v-radio>
+              </v-radio-group>
+            </v-col>
+          </v-row>
+          <v-row v-if="newRequestModel.contactMethod == '2'" no-gutters class="mt-2">
+            <v-col class="v-col-12 v-col-md-3 v-col-xl-2 blueText pt-3">
+              <strong>Business phone:</strong>
+            </v-col>
+            <v-col class="v-col-12 v-col-md-9 v-col-xl-10">
+              <v-text-field v-model="newRequestModel.phone" variant="outlined" :rules="[...rules.required, rules.phone]" />
+            </v-col>
+          </v-row>
+        </v-form>
+      </template>
+      <template #button>
+        <v-row justify="space-around">
+          <AppButton id="cancel-new-request" :isPrimary="false" size="large" width="200px" @click="closeNewRequestDialog()" :loading="isLoading">Cancel</AppButton>
+          <AppButton id="submit-new-request" size="large" width="200px" @click="submitNewRequest()" :loading="isLoading">Submit</AppButton>
+        </v-row>
+      </template>
+    </AppDialog>
+    <NewRequestConfirmationDialog
+      :referenceNumber="referenceNumber"
+      :showNewRequestConfirmationDialog="showNewRequestConfirmationDialog"
+      @closeNewRequestConfirmationDialog="closeNewRequestConfirmationDialog" />
+  </v-container>
+</template>
+
+<script>
+import { mapState, mapActions } from 'pinia'
+import { useAuthStore } from '@/stores/auth'
+import { useAppStore } from '@/stores/app'
+import { useMessagesStore } from '@/stores/messages'
+import AppButton from '../ui/AppButton.vue'
+import AppDialog from '../ui/AppDialog.vue'
+import rules from '@/utils/rules'
+import NewRequestConfirmationDialog from '@/components/messages/NewRequestConfirmationDialog.vue'
+
+export default {
+  name: 'NewRequestDialog',
+  components: { AppButton, AppDialog, NewRequestConfirmationDialog },
+  props: {
+    showNewRequestDialog: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  emits: ['closeNewRequestDialog'],
+  data() {
+    return {
+      dialogTitle: 'New request',
```

Comment:
This doesn't need to be part of data() as it's not reactive. Just put the text directly in.